instrument.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+
+# validate input
+case "$DISTRIBUTION" in
+  "linux-glibc"|"linux-musl"|"macos"|"windows")
+    ;;
+  *)
+    echo "Please specify the distribution by setting the DISTRIBUTION env var. Supported values: linux-glibc, linux-musl, macos, windows." >&2
+    return 2
+    ;;
+esac
+
+case "$ENABLE_PROFILING" in
+  "true"|"false")
+    ;;
+  "")
+    ENABLE_PROFILING="true"
+    ;;
+  *)
+    echo "Invalid ENABLE_PROFILING env var. Supported values: true, false." >&2
+    return 2
+    ;;
+esac
+
+# set defaults
+test -z "$INSTALL_DIR" && INSTALL_DIR="./otel-dotnet-auto"
+
+
+test -z "$ENABLE_PROFILING" && ENABLE_PROFILING="1"
```

Comment:
I think it is worth a comment why this is necessary.

Reply:
it is not :) removing

Reply:
U